widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -166,20 +167,43 @@ class ChatController(
     @Volatile
     private var chatState: ChatState
 
-    fun initChat(companyName: String?, queueId: String?, visitorContextAssetId: String?, chatType: ChatType) {
+    private val isSecureEngagement by lazy { isSecureEngagementUseCase() }
```

Comment:
Not sure about it. I made changes and according to them the result of the use case can changing.

Reply:
I'll turn it to getter